pkg/policy/origin.go

```diff
@@ -0,0 +1,22 @@
+package policy
+
+import (
+	"net/mail"
+)
+
+// Origin represents a potential email origin, allows policies for it to be queried.
+type Origin struct {
+	mail.Address
+	addrPolicy *Addressing
+	// LocalPart is the part of the address before @, including +extension.
+	LocalPart string
+	// Domain is the part of the address after @.
+	Domain string
+	// Mailbox is the canonical mailbox name for this recipient.
+	Mailbox string
```

Comment:
I don't think Mailbox will needed for Origin, it's referring to the Inbucket mailbox name.

Reply:
Done